libsodium-sys/build.rs

```diff
@@ -140,7 +141,7 @@ fn main() {
         }
 
         // Run `make check`, or `make all` if we're cross-compiling
-        let j_arg = format!("-j{}", unwrap!(env::var("NUM_JOBS")));
+        let j_arg = format!("-j{}", num_cpus::get());
```

Comment:
Interesting! I think ideally this should use [jobserver](http://make.mad-scientist.net/papers/jobserver-implementation/) from Cargo, to ensure that there are `n` jobs across the whole compilation graph, and to avoid the problem where each of n jobs spawns another n jobs. 

This is how to supply make with jobserver information: https://docs.rs/jobserver/0.1.8/jobserver/#examples. 

Obviously, this is a minor improvement which is completely unrelated to changes in this PR.